wordfreq_builder/wordfreq_builder/word_counts.py

```diff
@@ -67,12 +67,16 @@ def freqs_to_cBpack(in_filename, out_filename, cutoff=-600, lang=None):
 
     Only words with a frequency greater than `cutoff` centibels will be
     written to the new file.
+
+    This cutoff should not be stacked with a cutoff in `read_freqs`; doing
+    so would skew the resulting frequencies.
     """
-    freq_cutoff = 10 ** (cutoff / 100.)
-    freqs = read_freqs(in_filename, freq_cutoff, lang=lang)
+    freqs = read_freqs(in_filename, cutoff=0, lang=lang)
     cBpack = []
     for token, freq in freqs.items():
         cB = round(math.log10(freq) * 100)
+        if cB <= cutoff:
+            continue
```

Comment:
This is `continue` here and `break` above.  Is this one expecting unordered input?